### PR TITLE
One

### PR DESCRIPTION
**Pull request by peterennis**
Fix https://github.com/ModusCreateOrg/stop-watch-box/issues/9

stop-watch-2

### CODE REVIEW

**Comment by grgur** [comment]
Thanks for the update @peterennis

File: src/components.d.ts [hunk 1]
```diff
@@ -5,77 +5,67 @@
...
     'seconds': string;
   }
-  interface StopWatchAttributes extends StencilHTMLAttributes {
-    'applyHover'?: boolean;
-    'hours'?: string;
-    'milliseconds'?: string;
-    'minutes'?: string;
-    'seconds'?: string;
+  interface StopWatchBox {
+    'applyHover': boolean;
```
Comment (AhsanAyaz) [inline src/components.d.ts]: @peterennis is there a particular reason why we're making this property mandatory?

File: src/components/stop-watch-box/stop-watch-box.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Component, State, Prop } from "@stencil/core";
+import { h, Component, State, Prop } from "@stencil/core";
```
Comment (AhsanAyaz) [inline src/components/stop-watch-box/stop-watch-box.tsx]:
```suggestion
import { Component, State, Prop } from "@stencil/core";
```

@peterennis we're not using this new import h further in the code.

File: src/components/stop-watch/stop-watch.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Component, Prop } from "@stencil/core";
+import { h, Component, Prop } from "@stencil/core";
```
Comment (AhsanAyaz) [inline src/components/stop-watch/stop-watch.tsx]:
```suggestion
import { Component, Prop } from "@stencil/core";
```

@peterennis we're not using this new import `h` further in the code.

File: src/components.d.ts (hunk 1 above)
Comment (peterennis) [reply src/components.d.ts]: Hover is part of the original code. The demo shows one with hover and one without. If you want to change it then you could have a prop value. Not part of this PR.

File: src/components/stop-watch/stop-watch.tsx (hunk 3 above)
Comment (peterennis) [reply src/components/stop-watch/stop-watch.tsx]: h is now required for Stencil.
https://github.com/ionic-team/stencil/blob/master/BREAKING_CHANGES.md

**Pull request merged**